base.cfg

```diff
@@ -19,9 +19,10 @@ parts =
     sphinx-python
     deploy-to-heroku
     omelette
-develop = .
-sources-dir = extras
+# develop = .
+sources-dir = src
 auto-checkout =
+    plone.restapi
```

Comment:
This doesn't seem right to me (and also causes issues when testing this locally, `mr.developer: Directory name for existing package 'plone.restapi' differs. Expected 'plone.restapi'.`). Why would we check out `plone.restapi` _again_ in a source checkout of that very package?

Reply:
@lukasgraf tried to find the cause for the release. Reverted.